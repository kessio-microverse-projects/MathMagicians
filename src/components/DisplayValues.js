import React from 'react';
import propTypes from 'prop-types';

function DisplayValues(props) {
  const { total, operation, next } = props;
  return (
    <div>
      {total}
      {operation}
      {next}
    </div>
  );
}

export default DisplayValues;

DisplayValues.propTypes = {
  total: propTypes.number,
  operation: propTypes.string,
  next: propTypes.string,
};
DisplayValues.defaultProps = {
  total: 0,
  operation: '',
  next: '',
};
