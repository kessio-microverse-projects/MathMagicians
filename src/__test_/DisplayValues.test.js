import React from 'react';
import renderer from 'react-test-renderer';
import DisplayValues from '../components/DisplayValues';

describe('snapshot test for DisplayValues Component', () => {
it('Matches the snapshot', () => {
    const tree = renderer.create(<DisplayValues />).toJSON();
    expect(tree).toMatchSnapshot('DisplayVlues');
})
});