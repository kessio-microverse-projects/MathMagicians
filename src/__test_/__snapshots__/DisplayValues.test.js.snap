// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<div>
  0
</div>
`;

exports[`snapshot test for DisplayValues Component Matches the snapshot 1`] = `
<div>
  0
</div>
`;

exports[`snapshot test for DisplayValues Component Matches the snapshot: Display 1`] = `
<div>
  0
</div>
`;

exports[`snapshot test for DisplayValues Component Matches the snapshot: DisplayVlues 1`] = `
<div>
  0
</div>
`;
